### ISSUE TITLE
Creating 2D from Ifc file.. With AutoDesk NavisWork Api

### ISSUE THREAD

**Issue by umarone** [Autodesk/revit-ifc#575]
Hi Everyone,
                         I want to ask you that can we convert 2D or 3D file from IFC file in AutoDesk NavisWork Apis available for dot net c#.
if anyone can help me with any code example. I am waiting for this...

**Comment by mrjpeterj** [comment]
You want to load a IFC file into Navisworks using the API ?

**Comment by umarone** [comment]
> You want to load a IFC file into Navisworks using the API ?

Yes sir, Basically I have an ifc file which is export from Autocad , basically i want to convert that file back to 2d means in dwg format from Navisworks using API.
I am using c# with Naviseworks ,  so can I do that. please if you can help me

**Comment by mrjpeterj** [comment]
Now it sounds like you are asking whether you can export from Navisworks to a DWG format.
Navisworks cannot do this, whether in the UI or with the API
